packages/docregistry/src/context.ts

```diff
@@ -598,6 +601,8 @@ class Context<T extends DocumentRegistry.IModel> implements DocumentRegistry.ICo
         return this._manager.contents.save(this._path, options);
       } else if (result.button.label === 'REVERT') {
         return this.revert().then(() => { return model; });
+      } else if (result.button.label === 'CANCEL') {
```

Comment:
I propose we remove these two `else`s because they are superfluous since the `if`s all return.